FF142 Expr/Relnote Integrity-Policy (#40671)
mdn/content: files/en-us/mozilla/firefox/releases/142/index.md

  The CSS {{CSSXRef("anchor-size")}} function enables setting anchor-positioned element's size, position, and margins relative to the dimensions of anchor elements. ([Firefox bug 1972610](https://bugzil.la/1972610)).

- **`Integrity-Policy` and `Integrity-Policy-Report-Only`** (Nightly): `security.integrity_policy.enabled`

>   The {{httpheader("Integrity-Policy")}} and {{httpheader("Integrity-Policy-Report-Only")}} HTTP headers are now supported, allowing websites to enforce [subresource integrity guarantees](/en-US/docs/Web/Security/Subresource_Integrity) for scripts, or just report on violations of the policy, respectively.

Author: ```suggestion
  The {{httpheader("Integrity-Policy")}} and {{httpheader("Integrity-Policy-Report-Only")}} HTTP headers are now supported. These allow websites to either enforce [subresource integrity guarantees](/en-US/docs/Web/Security/Subresource_Integrity) for scripts or only report violations of the policy, respectively.
```
